### DISCUSSION TITLE
Paste clipboard content into a text input via keyboard shortcut

### DISCUSSION THREAD

**Discussion by scottopherson** [go-rod/rod#790] [Q&A]
Does anyone have a working example of pasting clipboard content into an input element using keyboard shortcut?

I've tried the following within a macos environment:

```go
textarea.MustKeyActions().Press(input.MetaLeft).Type(input.KeyV).MustDo()

textarea.MustKeyActions().Press(input.ControlLeft).Type(input.KeyV).MustDo()
```

and I've also tried various other combinations of `Press` and `Type` but have not had any luck yet on getting clipboard content to paste into an input element.

In Playwright, I currently achieve this with:

```ts
await textarea.press("Meta+V");
```

My current use-case is to simulate a user who: navigates to a website, clicks a "copy to clipboard" button, and then pastes the content into another website via keyboard shortcut.

**Comment by ysmood** [comment]
I think you need to use this:

https://github.com/go-rod/rod/blob/938f47b76cd93d338d559494f98b90317b987293/lib/input/mac_comands.go#L122

**Reply by scottopherson** [reply]
I found that file but not sure what to do with `macCommands`. Pass one of its values to `Type` or `Press`? I found no examples of its usage. Also for some reason `macCommands` is not available when I import `input` package. Only the `IsMac` var is available.
